cockatrice/src/window_main.cpp

```diff
@@ -296,6 +297,9 @@ void MainWindow::serverTimeout()
 void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32 endTime)
 {
     switch (r) {
+        case Response::RespClientUpdateRequired:
+            QMessageBox::critical(this, tr("Error"), tr("Your client does not contain features that the server requires, please update your client and try again."));
```

Comment:
Are you certain you want to do this?  It sounds ok for maybe a few missing features but if you get a client that is WAY out of date your message box will be long and ugly.